README.md

```diff
@@ -20,12 +20,18 @@ Future features include the following:
 
 To use Kubedash, the following are required:
 - A Kubernetes cluster of v1.0 or higher is operational and accessible from kubectl.
-- Heapster v0.18.0 is running as the `heapster` service on the `kube-system` namespace.
+- For Kubernetes <1.2 Heapster v0.18.0 is running as the `heapster` service on the `kube-system` namespace.
 
 After cloning this repository, use the following command to create the Kubedash pod:
 
+For Kubernets 1.0 and 1.1:
+
 `kubectl create -f deploy/kube-config.yaml`
 
+For Kubernetes 1.2 and higher:
+
+`kubectl create -f deploy/bundle.yaml`
```

Comment:
:+1: While reviewing on phone, I overlooked the fact that this PR was in the kubedash repo.. Sorry for all the confusion..